Configurations/unix-Makefile.tmpl

```diff
@@ -1024,7 +1100,7 @@ EOF
 $target: $deps
 	\$(CC) \$(LIB_CFLAGS) $linkflags\$(LIB_LDFLAGS)$shared_soname$shared_imp \\
 		-o $target_full$shared_def $objs \\
-                \$(PLIB_LDFLAGS) $linklibs \$(EX_LIBS)
+                \$(PLIB_LDFLAGS) $linklibs \$(LIB_EX_LIBS)
```

Comment:
I don't see that PLIB_LDFLAGS is assigned anywhere... And there are no plib_lfags left in Configurations/*.conf. Note that it was -Wl,-search_paths_first and original idea behind plib was place it in specific place on command line... In other words this needs double-checking on MacOS X, and in worst case changes would need to be reverted...

Reply:
Ah, right, need to clean that too.  It should get passed via `LIB_LDFLAGS`, so `PLIB_LDFLAGS` has become irrelevant,  I do suspect that the exact position of `-Wl,-search_paths_first` isn't crucial as long as it happens before all the -L and -l arguments...